content/en/docs/setup/_index.md

```diff
@@ -100,6 +100,7 @@ The following production environment solutions table lists the providers and the
 | [OpenNebula](https://www.opennebula.org)          |[OpenNebula Kubernetes](https://marketplace.opennebula.systems/docs/service/kubernetes.html)              |        |            |      |   |
 | [OpenShift](https://www.openshift.com)          |[OpenShift Dedicated](https://www.openshift.com/products/dedicated/) and [OpenShift Online](https://www.openshift.com/products/online/)              |        | [OpenShift Container Platform](https://www.openshift.com/products/container-platform/)            |      |  [OpenShift Container Platform](https://www.openshift.com/products/container-platform/)        |[OpenShift Container Platform](https://www.openshift.com/products/container-platform/)
 | [Oracle Cloud Infrastructure Container Engine for Kubernetes (OKE)](https://docs.cloud.oracle.com/iaas/Content/ContEng/Concepts/contengoverview.htm)          | &#x2714;             |   &#x2714;     |             |      |          |
+ [OVHcloud](https://www.ovhcloud.com/en/public-cloud/)         | [Managed Kubernetes Service](https://www.ovhcloud.com/en/public-cloud/kubernetes/)              | [OVHcloud Public Cloud](https://www.ovhcloud.com/en/public-cloud/)        |             |     | &#x2714;          |
```

Comment:
This seems to be missing a leading `|`. Also, the name Managed Kubernetes Service seems very generic; it it in fact “OVHcloud Managed Kubernetes Service”?

In the future it's likely that this entry will be removed, along with others, following policy changes under discussion in https://github.com/kubernetes/enhancements/pull/1327

Reply:
Indeed, was missing the initiatl pipe, thanks for the catch.
Now corrected. We went indeed for a descriptive name, as most of the industry.